.travis.yml

```diff
@@ -10,7 +10,7 @@ before_install:
 env:
   - RELEASE=../ga/23.0.0.9
```

Comment:
The 23.0.0.9 directory must be removed. We only support the last 2 quarterly releases (24.0.0.3 and 23.0.0.12)